scripts/upgrade.sh

```
@@ -5645,7 +5646,7 @@ prepare_hard_cut_target_controller() {
         || die "uv not found on PATH — cannot prepare the fresh target controller. No services changed."
     [[ -x "${DEFENSECLAW_VENV}/bin/python" ]] \
         || die "The installed bridge Python environment is unavailable. No services changed."
-    base_python="$(${DEFENSECLAW_VENV}/bin/python -I -c \
+    base_python="$(${DEFENSECLAW_VENV}/bin/python -I -B -c \
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)